programs/locker/src/constants/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod transfer_memo;
```

Comment:
may be wrap all constants in a file, we dont need a new module here (only 2 constants)

Reply:
updated